### ISSUE TITLE
Hide apps doesn't work properly

### ISSUE THREAD

**Issue by neolovich** [michelelacorte/FlickLauncher#35]
With the last update to 152 the function to hide apps improved. But at least there are a few apps which cant be hidden. And in the opposite there are also (in my case) at least one app which is hidden without that i marked it at hidden.
I think you are working on it. So take it just as a feedback. I'm not complaining.
By the way the option to turn off the page indicator is gone with the last update.

**Comment by michelelacorte** [comment]
Hi, thanks for reporting, I will check it better, for page indicators I've removed option to remove it because it will only restore default!

**Comment by neolovich** [comment]
But you add the function again in an future update?

**Comment by michelelacorte** [comment]
Maybe in future update, otherwise which app can't hide?

**Comment by neolovich** [comment]
That are a dictionary App, called "Duden", and the stock dialer and contact app. The dialer and the contact App which i installed additional are hidden although i didnt marked them.

**Comment by michelelacorte** [comment]
Mmm... maybe because it has the same package name, com.android.contacts

**Comment by neolovich** [comment]
Yes could be. Because i also use two different dictionaries named "Duden".

**Comment by neolovich** [comment]
You are quick, thank you.
For the dictionary "Duden" it works now. For the stock dialer/contacts and the dialer/contacts i installed it doesnt work. I select one and after i restart the Launcher there are both apps hidden and marked.

**Comment by neolovich** [comment]
For me it doesnt matter because i have the dialer on the homescreen. Thank you!
